dataeng/resources/load-google-spreadsheet-snowflake.sh

```diff
@@ -4,5 +4,10 @@ env
 ${WORKSPACE}/analytics-configuration/automation/run-automated-task.sh \
  LoadGoogleSpreadsheetsToSnowflakeWorkflow --local-scheduler \
  --date $(date +%Y-%m-%d -d "$TO_DATE") \
+ --sf-credentials $SNOWFLAKE_CREDENTIALS \
+ --sf-warehouse $SNOWFLAKE_WAREHOUSE \
+ --sf-role $SNOWFLAKE_ROLE \
+ --sf-database $SNOWFLAKE_DATABASE \
```

Comment:
Where is this coming from?  And why is it passed if it is not defined in Luigi?

Reply:
That said, it would be nice in general to be able to override values that were specified in config files, rather than hardcoding the gets to only pull values from config files.  It may matter less for this specific workflow, but in general when testing or doing a copy from Vertica, one may want to point to a more ad-hoc location in Snowflake.  Like now, where I'm working on loading dbt validation datasets.   (That may need to be added to those jobs....) .  But just for the record....

Reply:
Hm, I thought I fixed this already, probably didn't add it to the git index.  Not sure how this even worked when I tested it.

Reply:
To your second point, I also do not like that we are using prod-edx.cfg for this job, but the intention of the ticket was to re-use the same "code" from the vertica job to minimize difficulties.  In retrospect this ticket was already so damn difficult that I could have just re-done the config.